lib/expired_record_cleaner.rb

```diff
@@ -0,0 +1,53 @@
+class ExpiredRecordCleaner
+  attr_reader :verbose, :fiscal_year_start
+
+  def initialize(dtim, ok_to_act = false, verbose = false)
+    @fiscal_year_start = self.class.fiscal_year_start(dtim)
+    @ok_to_act = ok_to_act
+    @verbose = verbose
+  end
+
+  def vacuum_old_proposals
+    proposals = Proposal.pending.where("created_at < '#{@fiscal_year_start}'")
+    ids = []
+    proposals.each do |proposal|
```

Comment:
I return the list of ids in order to make it easier to test:

https://github.com/18F/C2/pull/749/files#diff-f27118fd70df5786f0be909423181f14R16